sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -8845,12 +8844,12 @@
                   "key": "uploadSupportingDocuments",
                   "tags": [],
                   "properties": {},
-                  "customConditional": "",
+                  "customConditional": "show = data['bclivingSituation'];",
```

Comment:
Should it not be just based on the `Not a BC Resident Panel`. 

i.e as soon as No is answered then it must show up.

Reply:
I don't think so. I feel it might confuse the user if it shows up when they haven't selected any of the 4 radio buttons.

Reply:
Checked with @CarlyCotton on this. The expectation is the way it is currently implemented. Also follows the same from the additional transportation cost.

<img width="1151" alt="image" src="https://github.com/user-attachments/assets/ff564424-e90d-448e-8322-c78f04a226f2">

Reply:
Ok, I am good if that is accepted by business.

Reply:
> Should it not be just based on the `Not a BC Resident Panel`. ? please confirm.
> 
> i.e as soon as No is answered then it must show up.
> 
> So to achieve this technically, no condition is required here.

I agree with this point and I am not sure why was it a business decision.
The uploader is no longer dependent on which option was selected. The uploader is a required filed in the same way the options are.
I cannot see also how it would confuse the user.